pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -44,6 +45,7 @@ func (r *REST) Create(ctx genericapirequest.Context, obj runtime.Object, include
 	if !ok {
 		return nil, apierrors.NewBadRequest(fmt.Sprintf("not a TokenReview: %#v", obj))
 	}
+	validation.ValidateTokenReview(tokenReview)
```

Comment:
Thanks for your information! @liggitt 

I didn't realize that tokenReview is a non-persisted resource.  I agree checking the length inline.